rawterm.h

```diff
@@ -39,572 +39,528 @@
+				return {'Z', {rawterm::Mod::Shift}, raw};
+			case '\x5B':
+				return {'[', {}, raw};
+			case '\x5C':
+				return {'\\', {}, raw};
+			case '\x5D':
+				return {']', {}, raw};
+			case '\x5E':
+				return {'^', {}, raw};
+			case '\x5F':
+				return {'_', {}, raw};
+			case '\x60':
+				return {'`', {}, raw};
+			case '\x61':
+				// LOWERCASE LETTERS
+				return {'a', {}, raw};
+			case '\x62':
+				return {'b', {}, raw};
+			case '\x63':
+				return {'c', {}, raw};
+			case '\x64':
+				return {'d', {}, raw};
+			case '\x65':
+				return {'e', {}, raw};
+			case '\x66':
+				return {'f', {}, raw};
+			case '\x67':
+				return {'g', {}, raw};
+			case '\x68':
+				return {'h', {}, raw};
+			case '\x69':
+				return {'i', {}, raw};
+			case '\x6A':
+				return {'j', {}, raw};
+			case '\x6B':
+				return {'k', {}, raw};
+			case '\x6C':
+				return {'l', {}, raw};
+			case '\x6D':
+				return {'m', {}, raw};
+			case '\x6E':
+				return {'n', {}, raw};
+			case '\x6F':
+				return {'o', {}, raw};
+			case '\x70':
+				return {'p', {}, raw};
+			case '\x71':
+				return {'q', {}, raw};
+			case '\x72':
+				return {'r', {}, raw};
+			case '\x73':
+				return {'s', {}, raw};
+			case '\x74':
+				return {'t', {}, raw};
+			case '\x75':
+				return {'u', {}, raw};
+			case '\x76':
+				return {'v', {}, raw};
+			case '\x77':
+				return {'w', {}, raw};
+			case '\x78':
+				return {'x', {}, raw};
+			case '\x79':
+				return {'y', {}, raw};
+			case '\x7A':
+				return {'z', {}, raw};
+			case '\x7B':
+				return {'{', {}, raw};
+			case '\x7C':
+				return {'|', {}, raw};
+			case '\x7D':
+				return {'}', {}, raw};
+			case '\x7E':
+				return {'~', {}, raw};
+			case '\x7F':
+				// BACKSPACE
+				return {' ', {rawterm::Mod::Backspace}, raw};
+		}
+
+		return {' ', {rawterm::Mod::Unknown}, raw};
+	}
+
+	inline rawterm::Pos get_term_size() {
+		struct winsize w;
+		ioctl(0, TIOCGWINSZ, &w);
+		return Pos { w.ws_row, w.ws_col };
+	}
+
+	inline void clear_screen() {
+		std::cout << "\x1B[2J";
+	}
+
+	inline void move_cursor(rawterm::Pos pos) {
+		std::cout << "\x1B[" << std::to_string(pos.line) << ';' << std::to_string(pos.col) << 'H' << std::flush;
+	}
+
+	inline void save_cursor_position() {
+		std::cout << "\x1B[s" << std::flush;
+	}
+
+	inline void load_cursor_position() {
+		std::cout << "\x1B[u" << std::flush;
+	}
+
+	// https://stackoverflow.com/a/48449104
+	inline void cursor_block_blink() {
+		std::cout << "\1\x1B[1 q\2" << std::flush;
+	}
+
+	inline void cursor_block() {
+		std::cout << "\1\x1B[2 q\2" << std::flush;
+	}
+
+	inline void cursor_underscore_blink() {
+		std::cout << "\1\x1B[3 q\2" << std::flush;
+	}
+
+	inline void cursor_underscore() {
+		std::cout << "\1\x1B[4 q\2" << std::flush;
+	}
+
+	inline void cursor_pipe_blink() {
+		std::cout << "\1\x1B[5 q\2" << std::flush;
+	}
+
+	inline void cursor_pipe() {
+		std::cout << "\1\x1B[6 q\6" << std::flush;
+	}
+
+	constexpr std::string bold(const std::string& s) {
```

Comment:
With these text styling functions being `constexpr`, the test files don't build. Can we replace `constexpr` with `inline` on these